test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -182,14 +193,78 @@ class db_size_statistic : public statistic {
     int64_t _limit;
 };
 
+class postrun_statistic_check {
+    public:
+    explicit postrun_statistic_check(configuration *config)
+    {
+        const auto config_stats = config->get_list(POSTRUN_STATISTICS);
+        /*
+         * Each stat in the configuration is a colon separated list in the following format:
+         * - Stat name
```

Comment:
I think it would be better to write it this way:
`<stat_name>:<min_limit>:<max_limit>`

Reply:
a557112